docs/cudf/source/conf.py

```diff
@@ -169,9 +174,9 @@ def clean_all_xml_files(path):
 # built documents.
 #
 # The short X.Y version.
-version = "24.04"
+version = rapids.version.version_major_minor
```

Comment:
Can we get this information directly from `cudf.__version__`? I think that package must be importable to build the docs, and the generated docs will actually reflect the installed `cudf` package version rather than the repo's `VERSION` file contents.

Reply:
We might want to use `packaging.version.Version` to parse out the `major.minor` bits we need.

Like:
```
import cudf
import packaging.version

pkg_version = packaging.version.Version(cudf.__version__)
version = f"{pkg_version.major:02}.{pkg_version.minor:02}"
```

Reply:
Yes, this is the approach I had intended to switch us to and never got around to. I'd prefer to pull from the package rather than reading the file.

`importlib.metadata.version` may be the more modern/canonical way of doing this. There are some subtle differences between that and requesting `cudf.__version__`, but for our purposes they should generally be equivalent and the former inspects package metadata which ought to be more robust to changes we make in how we store the version in the package.